src/main/resources/public/js/app.js

```diff
@@ -53,6 +54,7 @@ function fire(row, column) {
         dataType: "json"
     });
 
+
     request.done(function( currModel ) {
         displayGameState(currModel);
         gameModel = currModel;
```

Comment:
I tried it here, but it would never trigger. I moved the code around to a few different spots that I thought it would work, but it wouldn't work correctly in any of them. 

displayGameState() was the only place that it functioned appropriately.